tests/SpecTests/PrimaryStepDownSpecTest.php

```diff
@@ -215,7 +217,20 @@ public function testGetMoreIteration()
 
         // Send a {replSetStepDown: 5, force: true} command to the current primary and verify that the command succeeded
         $primary = $this->client->getManager()->selectServer(new ReadPreference(ReadPreference::RP_PRIMARY));
-        $primary->executeCommand('admin', new Command(['replSetStepDown' => 5, 'force' => true]));
+
+        $success = false;
+        $attempts = 0;
+        do {
+            try {
+                $attempts++;
+                $primary->executeCommand('admin', new Command(['replSetStepDown' => 5, 'force' => true]));
+                $success = true;
+            } catch (Exception $e) {
```

Comment:
Would it make sense to just use `DriverException` here? That's what's used in `waitForMasterReelection`, and I think this similar because we wouldn't expect a library exception from any of this logic.

Reply:
Absolutely correct. This is a driver exception, but PhpStorm decided to import the class even though `DriverException` was already imported. I've rectified this.